app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListPresenter.java

```diff
@@ -23,11 +23,7 @@
 
     @Override
     public void displayCategories() {
-        if (contestBuilder == null || contestBuilder.getCategories().size() == 0) {
-            view.showDefaultCategory();
```

Comment:
No actually not supposed to change the creation flow at all. The user should still be presented with an example/default category. It will just be done at a different step. I will rebase now to confirm.